dev_tools/conf/pip-list-dev-tools.txt

```diff
@@ -1,28 +1,27 @@
 # For testing and analyzing code.
-mypy~=0.701.0
-pylint~=2.3.0
-pytest~=3.8.0
-pytest-asyncio~=0.10.0
-pytest-cov~=2.5.0
-pytest-benchmark~=3.2.0
-yapf~=0.27.0
+mypy~=0.750
+pylint~=2.4
+pytest~=5.3
+pytest-asyncio~=0.10
+pytest-cov~=2.8
+pytest-benchmark~=3.2
+yapf~=0.29
 
 # For generating protobufs
-grpcio-tools~=1.24.0
-mypy-protobuf==1.10
+grpcio-tools~=1.25
+mypy-protobuf==1.16
 
 # For uploading packages to pypi.
 twine
 
 # For verifying behavior of qasm output.
-qiskit~=0.13.0
+qiskit~=0.13
```

Comment:
This change is wrong. It means that if qiskit makes a breaking change our build will break.